### PR TITLE
Added netrc support

### PR DESCRIPTION
**Pull request by raghunat**
POC in response to #16 :-)

### CODE REVIEW

File: src/terraform-provider-signalform/signalform/provider.go [hunk 1]
```diff
@@ -93,3 +101,51 @@ func readConfigFile(configPath string, config *signalformConfig) error {
...
+		return nil
+	}
+
+	// Load up the netrc file
+	net, err := netrc.ParseFile(path)
+	if err != nil {
+		return fmt.Errorf("Error parsing netrc file at %q: %s", path, err)
+	}
+
+	machine := net.FindMachine("signalfx")
```
Comment (bronislav) [inline src/terraform-provider-signalform/signalform/provider.go]: I'd use a full domain name to avoid confusion: `api.signalfx.com`

**Comment by bronislav** [comment]
@raghunat It works.

File: src/terraform-provider-signalform/signalform/provider.go (hunk 1 above)
Comment (poros) [reply src/terraform-provider-signalform/signalform/provider.go]: it seems reasonable to me

**Comment by raghunat** [comment]
@poros Updated, thanks!

**Comment by poros** [comment]
If @bronislav is fine with the PR too, I can merge and cut a release.

**Comment by bronislav** [comment]
@poros LGTM 👍

**Pull request merged**